applications/sintering/analysis_examples/49particles_delta_min_50_coarse.json

```diff
@@ -18,7 +18,7 @@
         "CustomBoundingBox": "false",
         "DivisionsPerInterface": "2",
         "HangingNodeWeight": "1",
-        "InterfaceBufferRatio": "3",
+        "InterfaceBufferRatio": "2.5",
```

Comment:
?

Reply:
This is acceptable. The distance factor is not obliged to be integer. In all those json files I tried to keep product `InterfaceBufferRatio*InterfaceWidth` more or less the same `=25`.